WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -4,20 +4,23 @@ Consider updating the versions in `WalletWasabi.Helpers.Constants`. If the versi
 
 ```sh
 sudo apt-get update && cd ~/WalletWasabi && git pull && cd ~/WalletWasabi/WalletWasabi.Backend && dotnet restore && cd ~
-sudo service nginx stop
 sudo systemctl stop walletwasabi.service
-sudo killall tor
 bitcoin-cli stop
 sudo apt-get upgrade -y && sudo apt-get autoremove -y
+# in case a new kernel is available:
 sudo reboot
+# service is restarted after boot, so stop it again
+sudo systemctl stop walletwasabi.service
```

Comment:
Yes. This is clearly better. However, given we don't have the `.service` files in place, this will change nothing.

Reply:
Hmm, the service file was mentioned below.